ci/test/06_script_b.sh

```diff
@@ -53,6 +62,7 @@ if [ "$CI_OS_NAME" == "macos" ]; then
   echo > "${HOME}/Library/Application Support/Bitcoin"
 else
   echo > "${HOME}/.bitcoin"
+  ${NONROOT_DUMMY_DIR}
```

Comment:
hm see https://github.com/bitcoin/bitcoin/pull/27850#discussion_r1239761122 not sure what to do

Reply:
re: https://github.com/bitcoin/bitcoin/pull/27850#discussion_r1262333855

> hm see [#27850 (comment)](https://github.com/bitcoin/bitcoin/pull/27850#discussion_r1239761122) not sure what to do

I don't actually understand what this issue is in that comment since the variable seems to be used just once.

It does seem strange to define these commands as variables instead of inlining them when the variables only seem to be used only once. Also would expect them to be shell functions instead of variables. And would expect them to have imperative names like NONROOT_MAKE_DUMMY_DIR instead of NONROOT_DUMMY_DIR. But I guess a nice thing about having the variables is it ties nonroot code scattered different places back to a central place. Anyway, whatever style you like to use here seems fine.